docs/adrs/00005-license-export-analytics.md

```diff
@@ -0,0 +1,126 @@
+For SPDX, the text field corresponds to the complete Expression. Similarly, the Expression will be parsed into a license array and stored in spdx_licenses, with parsing exceptions stored in spdx_license_exceptions.
+Currently missing SPDX's ExtractedLicensingInfos and SBOM-level licenses.
+
+### 4 License data structure in GUAC
+![erd_guac.png](erd_guac.png)
+GUAC parses all expressions into individual licenses and saves them separately. LicenseRef* represents ExtractedLicensingInfos, with its extractedText stored in the inline field.
+
+## Decision
+
+### Current design proposal
+![erd_license.png](erd_license.png)
+```rust
+pub enum LicenseCategory {
+    #[sea_orm(string_value = "slc")]
+    SPDXDECLARED,
+    #[sea_orm(string_value = "sld")]
+    SPDXCONCLUDED,
+    #[sea_orm(string_value = "clci")]
+    CYDXLCID,
+    #[sea_orm(string_value = "clcn")]
+    CYDXLCNAME,
+    #[sea_orm(string_value = "cle")]
+    CYDXLEXPRESSION,
+    #[sea_orm(string_value = "cd")]
+    CLEARLYDEFINED,
+    #[sea_orm(string_value = "o")]
+    OTHER,
+}
+```
+
+Currently, the system's licenses come from the following sources: 1) SPDX's declared license and concluded license, 2) CycloneDX, and 3) clearlyDefined. The license export only requires SPDX's declared license and CycloneDX licenses, while concluded licenses and clearlyDefined licenses are not needed. Therefore, the system must distinguish between them.
+
+SPDXDECLARED: The license from spdx sbom's declered license.
+SPDXCONCLUDED: The license from spdx sbom's concluded license.
+CYDXLCID: The license from CycloneDX sbom's id.
+CYDXLCNAME: The license from CycloneDX sbom's name.
+CYDXLEXPRESSION: The license from CycloneDX sbom's Expression.
+
+Where license_id is the same as name in GUAC, license_ref_id serves as a foreign key linking to the Id of extracted_licensing_infos. I illustrate this design through two unit tests https://github.com/trustification/trustify/pull/1164/files#diff-e2251167d81406b13fac64820ca3c5af49095705376eb2d6a21830270605d69cR99.
+This design currently has two issues:
+1. The complete expression is not saved, as the complete expression includes not only individual license
+   information but also the relationships between these licenses.
+2. SBOM-level licenses are not saved.
+
+### Next design proposal
+![next_erd_license.png](next_erd_license.png)
+
+
+The new design retains the original license structure, but adds a new attribute: license type. It also removes the physical association between the license and ExtractedLicensingInfos when the license is a licenseRef. If needed, we can retrieve this information through conditional queries. Additionally, a new entity, sbom_license, is introduced to store license information at the SBOM level. However, as I mentioned earlier, the SBOM-level licenses in SPDX and CycloneDX have completely different meanings, so a type attribute is required to distinguish between them.
```

Comment:
Design proposal should outline 'what we are going to do' ... useful context information should be in the context section.